composer.json

```diff
@@ -24,6 +24,6 @@
         "symfony/security-csrf": "~2.5",
         "symfony/twig-bridge":   "~2.5",
         "org_heigl/daterange": "~1.1",
-        "guzzlehttp/guzzle": "^6.1"
+        "guzzlehttp/guzzle": "~6.2.1"
```

Comment:
Why is this change here?
Unless, I've misunderstood, ~6.2.1 means that we won't pick up 6.3.0.